src/components/buttons/button-component/button.component.tsx

```diff
@@ -6,8 +6,17 @@ import React, { Ref } from "react";
 import styled from "styled-components";
 import { fontFamilyPrimary, colors } from "../../../assets/colors";
 
+export type MarginString =
```

Comment:
To było to `TODO`
Dodałem możliwość zmiany marginesów na przyciskach

![image](https://user-images.githubusercontent.com/21079319/120377860-f82b7e80-c31d-11eb-8e18-6a534b5dfd7c.png)